src/ProductlineApp.WebUI/ClientApp/src/components/atoms/buttons/cancelButton/CancelButton.tsx

```diff
@@ -1,10 +1,10 @@
 import { Link } from 'react-router-dom';
 import './css/cancelButton.css';
 
-function CancelButton({ pathTo, close }: { pathTo?: any; close?: any }) {
+function CancelButton({ pathTo, close, setShowButtons, showButtons }: { setShowButtons?: any, showButtons?: boolean; pathTo?: any; close?: any }) {
   return (
-    <Link to={pathTo} className="returnToProductsLink" id="link">
-      <div className="cancelButton" onClick={close}>
+    <Link to={pathTo} className="returnToProductsLink" id="link" >
+      <div className="cancelButton" onClick={() => { setShowButtons(showButtons); close }}>
```

Comment:
lepiej by bylo przekazac tylko referencje do funkcji z komponentu nadrzednego niz wywolywac to tutaj. Wystarczyloby w argumentach funkcji dac metode "onClick", ktora sie przekaze tutaj - dzieki temu tez nie trzeba by bylo przekazywac "showButtons", tylko obsluzone by to bylo w komponencie nadrzednym.